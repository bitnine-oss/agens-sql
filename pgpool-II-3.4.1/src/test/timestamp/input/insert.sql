INSERT INTO rel1 DEFAULT VALUES
INSERT INTO rel2 DEFAULT VALUES
INSERT INTO rel2(c1) VALUES(1)
INSERT INTO rel1 VALUES(1, now(), 2, CURRENT_DATE)
INSERT INTO rel1 VALUES(3, CURRENT_TIMESTAMP, 4, CURRENT_TIME)
INSERT INTO rel1 VALUES(5, CURRENT_TIMESTAMP(0), 6, CURRENT_TIME(0))
INSERT INTO rel1 VALUES(7, LOCALTIMESTAMP, 8, LOCALTIME)
INSERT INTO rel1 VALUES(9, LOCALTIMESTAMP(0), 10, LOCALTIME(0))
INSERT INTO rel1 VALUES(11, DEFAULT);
INSERT INTO rel1(c3) VALUES(1)
INSERT INTO rel1(c2, c1) VALUES('2000-1-1', 1)
INSERT INTO rel1(c2, c1) VALUES(DEFAULT, 2)
INSERT INTO rel1(c2, c1) VALUES(DEFAULT, 3), (DEFAULT, 4), ('2009-1-1', 5)
INSERT INTO rel1(c1, c2, c4) VALUES(1, '2009-1-1', '2009-2-2')
PREPARE q AS INSERT INTO rel1(c3) VALUES(1)
PREPARE q(int) AS INSERT INTO rel1(c3) VALUES($1)
