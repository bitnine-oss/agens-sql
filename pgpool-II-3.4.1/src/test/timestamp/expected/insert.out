INSERT INTO "rel1" VALUES (DEFAULT,'2009-01-01 23:59:59.123456+09',DEFAULT,'2009-01-01 23:59:59.123456+09')
INSERT INTO rel2 DEFAULT VALUES
INSERT INTO rel2(c1) VALUES(1)
INSERT INTO "rel1" VALUES (1,"pg_catalog"."timestamptz"('2009-01-01 23:59:59.123456+09'::text),2,'2009-01-01 23:59:59.123456+09'::text::date)
INSERT INTO "rel1" VALUES (3,"pg_catalog"."timestamptz"('2009-01-01 23:59:59.123456+09'::text),4,'2009-01-01 23:59:59.123456+09'::text::timetz)
INSERT INTO "rel1" VALUES (5,'2009-01-01 23:59:59.123456+09'::text::timestamptz(0),6,'2009-01-01 23:59:59.123456+09'::text::timetz(0))
INSERT INTO "rel1" VALUES (7,'2009-01-01 23:59:59.123456+09'::text::timestamp,8,'2009-01-01 23:59:59.123456+09'::text::time)
INSERT INTO "rel1" VALUES (9,'2009-01-01 23:59:59.123456+09'::text::timestamp(0),10,'2009-01-01 23:59:59.123456+09'::text::time(0))
INSERT INTO "rel1" VALUES (11,'2009-01-01 23:59:59.123456+09',DEFAULT,'2009-01-01 23:59:59.123456+09')
INSERT INTO "rel1"("c3", "c2", "c4") VALUES (1,'2009-01-01 23:59:59.123456+09','2009-01-01 23:59:59.123456+09')
INSERT INTO "rel1"("c2", "c1", "c4") VALUES ('2000-1-1',1,'2009-01-01 23:59:59.123456+09')
INSERT INTO "rel1"("c2", "c1", "c4") VALUES ('2009-01-01 23:59:59.123456+09',2,'2009-01-01 23:59:59.123456+09')
INSERT INTO "rel1"("c2", "c1", "c4") VALUES ('2009-01-01 23:59:59.123456+09',3,'2009-01-01 23:59:59.123456+09'), ('2009-01-01 23:59:59.123456+09',4,'2009-01-01 23:59:59.123456+09'), ('2009-1-1',5,'2009-01-01 23:59:59.123456+09')
INSERT INTO rel1(c1, c2, c4) VALUES(1, '2009-1-1', '2009-2-2')
PREPARE "q" (timestamptz,timestamptz) AS INSERT INTO "rel1"("c3", "c2", "c4") VALUES (1,$1,$2)
PREPARE "q" (int4,timestamptz,timestamptz) AS INSERT INTO "rel1"("c3", "c2", "c4") VALUES ($1,$2,$3)
